hbw/app/controllers/hbw/users_controller.rb

```diff
@@ -10,6 +10,10 @@ def lookup
       render :index
     end
 
+    def check_user
+      render json: {user_exist: widget.user_exist?(current_user_identifier)}.to_json
```

Comment:
`.to_json` is already called internally, just remove it
also, please, use spaces on both sides of curly brackets, it's the silent style guide of this project (see https://github.com/bbatsov/ruby-style-guide#spaces-braces for details)